cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -22,6 +22,19 @@ ARTIFACT_DOWNLOAD_URL="${INSTALLER_DOWNLOAD_URL:-{{ base_url }}}"
 PRINT_VERBOSE=${INSTALLER_PRINT_VERBOSE:-0}
 PRINT_QUIET=${INSTALLER_PRINT_QUIET:-0}
 NO_MODIFY_PATH=${INSTALLER_NO_MODIFY_PATH:-0}
+ALLOW_RECEIPTS_TO_FAIL=0
```

Comment:
I don't think we need/want to add this -- certainly the powershell installer doesn't so this is just making it more inconsistent?

Reply:
Yeah, this isn't properly used anymore so I'll take it out.